Test/RachelTest.cs

```diff
@@ -0,0 +1,76 @@
+﻿using EngineLayer;
+using MassSpectrometry;
+using MzLibUtil;
+using NUnit.Framework;
+using Proteomics;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using TaskLayer;
+using UsefulProteomicsDatabases;
+
+namespace Test
+{
+    [TestFixture]
+    public class RachelTest
```

Comment:
While that is true, i t is helpful for organizing the tests. Something like PruningTests. Otherwise we might as well toss all the tests into a single class.

Reply:
There are already pruning test methods within other test classes. I feel that it would be misleading to call the class this. It also appears that other tests would be difficult to migrate all into one class because the methods are dependent on other portions of the class.

Reply:
I am not advocating for a single test class. If other pruning test classes exist, can your methods be moved into those instead of generating a new class? It doesn't make sense for us to generate a new class every time a new person generates a unit test.

Reply:
You are welcome to ignore me, because "RobTest" has set a precedent, but I think better organization will help us from having an overwhelming number of test classes.

Reply:
I was able to group the pruned Db tests together with minimal effort

Reply:
Thanks for doing that! The class was renamed, but the .cs file is still RachelTest.cs. Can you change the file name to match? gptmdPrunedDbTests.cs? Also capitol "G"ptmd on both of those.